src/precice/impl/Participant.cpp

```diff
@@ -48,9 +48,9 @@ void Participant::addAction(action::PtrAction &&action)
   _actions.push_back(std::move(action));
 }
 
-void Participant::setUseMaster(bool useMaster)
+void Participant::setUsePrimaryRank(bool usePrimaryRank)
 {
-  _useMaster = useMaster;
+  _usePrimaryRank = usePrimaryRank;
```

Comment:
This one (and below) we should probably rename. It's probably a `useIntraComm` or maybe even `isParallel`?